content/doc/book/pipeline/syntax.adoc

```diff
@@ -53,42 +39,31 @@ pipeline {
 }
 ----
 
-The basic statements and expressions which are valid in Declarative Pipeline
-follow the same rules as
-link:http://groovy-lang.org/syntax.html[Groovy's syntax]
-with the following exceptions:
+The basic statements and expressions which are valid in Declarative Pipeline follow the same rules as link:http://groovy-lang.org/syntax.html[Groovy's syntax] with the following exceptions:
 
 * The top-level of the Pipeline must be a _block_, specifically: `pipeline { }`.
-* No semicolons as statement separators. Each statement has to be on its own
-  line.
-* Blocks must only consist of <<declarative-sections>>,
-  <<declarative-directives>>, <<declarative-steps>>, or assignment statements.
-* A property reference statement is treated as a no-argument method invocation. So, for
-  example, `input` is treated as `input()`.
-
-You can use the
-link:../getting-started/#directive-generator[Declarative Directive Generator]
-to help you get started with configuring the directives and sections in your
-Declarative Pipeline.
+* No semicolons as statement separators.
+Each statement has to be on its own line.
+* Blocks must only consist of <<declarative-sections>>, <<declarative-directives>>, <<declarative-steps>>, or assignment statements.
+* A property reference statement is treated as a no-argument method invocation.
+So, for example, `input` is treated as `input()`.
+
+You can use the link:../getting-started/#directive-generator[Declarative Directive Generator] to help you get started with configuring the directives and sections in your Declarative Pipeline.
 
 === Limitations
 
-There is currently an link:https://issues.jenkins.io/browse/JENKINS-37984[open issue] 
-which limits the maximum size of the code within the `pipeline{}` block. This limitation 
-does not apply to Scripted pipelines.
+There is currently an link:https://issues.jenkins.io/browse/JENKINS-37984[open issue]  which limits the maximum size of the code within the `pipeline{}` block.
+This limitation does not apply to Scripted pipelines.
```

Comment:
Correcting an error in the original.  Pipeline is supposed to always be capitalized (contrary to usual English language rules of capitalization)
```suggestion
This limitation does not apply to Scripted Pipelines.
```